lib/fastlane/actions/gcovr.rb

```diff
@@ -0,0 +1,91 @@
+module Fastlane
+  module Actions
+
+    # --object-directory=OBJDIR      Specify the directory that contains the gcov data files.
+    # -o OUTPUT, --output=OUTPUT     Print output to this filename Keep the temporary *.gcov files generated by gcov.
+    # -k, --keep                     Keep the temporary *.gcov files generated by gcov.
+    # -d, --delete                   Delete the coverage files after they are processed.
+    # -f FILTER, --filter=FILTER     Keep only the data files that match this regular expression
+    # -e EXCLUDE, --exclude=EXCLUDE  Exclude data files that match this regular expression
+    # --gcov-filter=GCOV_FILTER      Keep only gcov data files that match this regular expression
+    # --gcov-exclude=GCOV_EXCLUDE    Exclude gcov data files that match this regular expression
+    # -r ROOT, --root=ROOT           Defines the root directory for source files.
+    # -x, --xml                      Generate XML instead of the normal tabular output.
+    # --xml-pretty                   Generate pretty XML instead of the normal dense format.
+    # --html                         Generate HTML instead of the normal tabular output.
+    # --html-details                 Generate HTML output for source file coverage.
+    # --html-absolute-paths          Set the paths in the HTML report to be absolute instead of relative
+    # -b, --branches                 Tabulate the branch coverage instead of the line coverage.
+    # -u, --sort-uncovered           Sort entries by increasing number of uncovered lines.
+    # -p, --sort-percentage          Sort entries by decreasing percentage of covered lines.
+    # --gcov-executable=GCOV_CMD     Defines the name/path to the gcov executable [defaults to the GCOV environment variable, if present; else 'gcov'].
+    # --exclude-unreachable-branches Exclude from coverage branches which are marked to be excluded by LCOV/GCOV markers or are determined to be from lines containing only compiler-generated "dead" code.
+    # -g, --use-gcov-files           Use preprocessed gcov files for analysis.
+    # -s, --print-summary            Prints a small report to stdout with line & branch percentage coverage
+
+    class GcovrAction
+      ARGS_MAP = {
+        object_directory: '--object-directory',
+        output: '-o',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.